vassal-app/src/main/java/VASSAL/counters/Labeler.java

```diff
@@ -618,7 +618,8 @@ public BufferedImage eval() throws Exception {
 
       // paint the foreground
       if (fg != null) {
-        if (txt.contains("<img")) { //NON-NLS
+        // Support images pre-existing in the module
+        if (txt.contains("<img") && !txt.contains("http://") && !txt.contains("https://") && !txt.contains("file://")) { //NON-NLS
```

Comment:
I think you might want to match `"src=\"http://"`, etc. just to be safe.

Reply:
On reflection, this would be best as a regex, to save checking the string multiple times.

Reply:
I think `<img ([^>]+ )? src="(https?|file)://` is the regex we want.

Reply:
done